providers/src/airflow/providers/celery/provider.yaml

```
@@ -330,6 +330,17 @@ config:
         type: string
         example: ~
         default: "False"
+      extra_celery_config:
+        description: |
+          Extra celery configs to include in the celery worker.
+          Any of the celery config can be added to this config and it
+          will be applied while starting the celery worker. e.g. {"worker_max_tasks_per_child": 10}
+          See also:
+          https://docs.celeryq.dev/en/stable/userguide/configuration.html#configuration-and-defaults
+        version_added: ~
+        type: string
+        example: ~
+        default: "{}"
```

Okay, I also did not know but defaults seem to be Format-String formatted and the string "{}" is an invalid sequence. So to effectively have an `{}` as default we need to use `{{}}` here to quote the brackets.
```suggestion
        default: "{{}}"
```

---

Okay ... interesting, this also resolves the CI problem - it seems the constrains also can not be built as the templating fails there. I did a PR with the fixes applied as test and this is turning green: https://github.com/apache/airflow/pull/45160
As with the other proposal a check is being made that the settings are a dict, you can even lave an empty string as default...
```suggestion
        default: ""
```